src/Conference/FavouriteConferenceList.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+const StyledFavouritesList = styled.ul`
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+`;
+
+const StyledFavouritesListItem = styled.li`
+  :hover {
+    background-color: papayawhip;
+    cursor: pointer;
+    display: inline;
+  }
+`;
+
+const StyledConferenceLink = styled(Link)`
+  display: inline-block;
+  text-decoration: none;
+  color: #322d38;
+`;
+
+const FavouriteConferenceList = favouriteConferences => {
```

Comment:
The first argument to a react component is the Props object, therefore you'll have to write it as ` ({ favouriteConferences })`